testcases/EMStress.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/python2
+# OpenPOWER Automated Test Project
+#
+# Contributors Listed Below - COPYRIGHT 2017
+# [+] International Business Machines Corp.
+#
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied. See the License for the specific language governing
+# permissions and limitations under the License.
+#
+
+import unittest
+import time
+import threading
+
+import OpTestConfiguration
+from common.OpTestSystem import OpSystemState
+from common.OpTestConstants import OpTestConstants as BMC_CONST
+from common.OpTestThread import OpSSHThreadLinearVar1, OpSSHThreadLinearVar2, OpSOLMonitorThread
+from testcases.OpTestEM import OpTestEM
+
+class EMTorture(unittest.TestCase, OpTestEM):
+    '''
+    Stress test for Energy management/OCC functionalities in FW/OPAL/Linux.
+    # tlbie test(IPI stress)
+    # CPU Hotplug torture
+    # Read frequency loop
+    # CPU idle states tests, only one idle state enabled at a time for certain duration
+    # CPU Governor change tests
+    # OCC Reset reload tests
+    '''
+    def setUp(self):
+        conf = OpTestConfiguration.conf
+        self.cv_IPMI = conf.ipmi()
+        self.cv_SYSTEM = conf.system()
+        self.host = conf.host()
+        self.torture_time = 780 # 12 hours
+        self.bmc_type = conf.args.bmc_type
+        self.thread_list = []
+        self.setup_test()
+
+    def runTest(self):
+        # tlbie test(IPI stress)
+        self.host.host_copy_tlbie_binary()
+        cmd = "/tmp/tlbie_test &"
+        self.host.host_run_command(cmd)
+        self.host.host_run_command(cmd)
+
+        # Read frequency
+        torture_time = self.torture_time
+        cmd_list = ['ppc64_cpu --frequency']
+        num = 1
+        thread = OpSSHThreadLinearVar1(num, "Thread-%s" % num, cmd_list, 2, torture_time, True)
+        thread.start()
+        self.thread_list.append(thread)
+
+        # CPU Governor change tests
+        torture_time = self.torture_time
```

Comment:
Append all threads into a list as you do already, and start all together later in a loop 
```
for threads in self.thread_list:
    thread.start()
```